bindings/generated/axelar-solana-gateway/src/coder/accounts.ts

```diff
@@ -1,7 +1,7 @@
 // @ts-nocheck
 import * as B from "@native-to-anchor/buffer-layout";
-import { AccountsCoder, Idl } from "@coral-xyz/anchor";
-import { IdlTypeDef } from "@coral-xyz/anchor/dist/cjs/idl";
+import { AccountsCoder, Idl } from "@project-serum/anchor";
+import { IdlTypeDef } from "@project-serum/anchor/dist/cjs/idl";
```

Comment:
Those changes came after I regenerated the bindings w/ `xtask create-bindings`. 

I'm not entirely sure about why the packages changed, but tests are passing, so I guess this is expected.